makefile

```diff
@@ -61,15 +61,17 @@ install:
 	@# install the xmake directory
 	@cp -r xmake/* $(xmake_dir_install)
 	@# make the xmake loader
+ifneq ($(PLAT),linux)
```

Comment:
What's this? Although we can get the executable path on linux, macOS and windows. But maybe we cannot get it successfully on some linux distributions.  So I think we still need `export XMAKE_PROGRAM_DIR=/xxx`